clean-ci-resources.sh

```diff
@@ -45,3 +45,4 @@ openstack container list -f value -c Name |\
 
 ./stale -q volume snapshot | xargs --no-run-if-empty openstack volume snapshot delete
 ./stale -q volume | xargs --no-run-if-empty openstack volume delete
+./stale -q floating ip | xargs --verbose --no-run-if-empty openstack floating ip delete
```

Comment:
I was under the impression that we looked for existing floating IPs before creating a new one in the CI jobs like we do in the [shiftstack-ci repo](https://github.com/shiftstack/shiftstack-ci/blob/master/run_ocp.sh#L17-L24), but that appears [to not be the case](https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml#L363-L364).
LGTM